app/page.tsx

```diff
@@ -1,3 +1,125 @@
-export default function Home() {
-  return <div></div>;
+"use client";
+import Image from "next/image";
+import { useState } from "react";
+import Input from "./components/Input";
+import Button from "./components/Button";
+import Letter from "./components/Icons/Letter";
+import Lock from "./components/Icons/Lock";
+import User from "./components/Icons/User";
+import LoadingSpinner from "./components/Icons/LoadingSpinner";
+import Confetti from "./components/Icons/Confetti";
+
+export default function SignUp() {
+  const [formData, setFormData] = useState({
+    email: "",
+    username: "",
+    password: "",
+    confirmPassword: "",
+  });
```

Comment:
take a look into [react-hook-forms](https://react-hook-form.com/)

Reply:
Follow @hyanez-lightit lead here.

Reply:
The idea is for this challenge to use this, and then on a separate PR implement` react-hook-form`